addons/storyshots/storyshots-core/src/frameworks/angular/loader.ts

```diff
@@ -17,7 +17,7 @@ function setupAngularJestPreset() {
   // is running inside jest -  one of the things that `jest-preset-angular/build/setupJest` does is
   // extending the `window.Reflect` with all the needed metadata functions, that are required
   // for emission of the TS decorations like 'design:paramtypes'
-  jest.requireActual('jest-preset-angular/build/setupJest');
+  jest.requireActual('jest-preset-angular/build/setup-jest');
```

Comment:
what about using a try-catch statement instead so that this would be backwards compatible with older jest-preset-angular versions?
```suggestion
  try {
    jest.requireActual('jest-preset-angular/build/setupJest');
  } catch(e) {
    jest.requireActual('jest-preset-angular/build/setup-jest');
  }
```

Reply:
this should then get around the current CI failure issue.

Reply:
why not. maybe @igor-dv or @thomasbertet can review that ?

Reply:
This looks like a reasonable workaround to me @rgunning